### ISSUE TITLE
Montecarlo dropout

### ISSUE THREAD

**Issue by fferroni** [open-mmlab/mmsegmentation#2441]
Hello,

I am using the 1.x version of the API. For the purposes of montecarlo dropout, how do i maintain dropout at inference time (so I can sample multiple times and take average of logits, rather than a single shot)?

Thanks!

**Comment by MeowZheng** [comment]
I suggest to rewrite the `TestLoop` in mmengine, as. the original `TestLoop` calls `model.eval()` by default 
https://github.com/open-mmlab/mmengine/blob/ad1b43faf2a9185e93ad37250aa11cfe5b6be33d/mmengine/runner/loops.py#L415

**Comment by MeowZheng** [comment]
This issue is inactive, and I will close it, if you still have any questions, just reopen it as your need.
